src/basic/cgroup-util.c

```diff
@@ -357,20 +357,26 @@ int cg_kill(
                 Set *s,
                 cg_kill_log_func_t log_kill,
                 void *userdata) {
-        int r;
+        int r, r2;
 
         r = cg_kill_items(controller, path, sig, flags, s, log_kill, userdata, "cgroup.procs");
         if (r < 0 || sig != SIGKILL)
                 return r;
 
+        r2 = r;
```

Comment:
Not a matter, but I'd like to rename `r2` -> `ret`, as in `cg_kill_item()` or `cg_kill_recursive()` we use `ret`.